x-pack/plugins/apm/server/routes/service_groups/get_apm_alerts_client.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APMRouteHandlerResources } from '../typings';
+
+export type ApmAlertsClient = Awaited<ReturnType<typeof getApmAlertsClient>>;
+
+export async function getApmAlertsClient({
+  plugins,
+  request,
+}: APMRouteHandlerResources) {
+  const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
+  const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(
+    request
+  );
+  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices([
+    'apm',
+  ]);
+
+  type ApmAlertsClientSearchParams = Omit<
+    Parameters<typeof alertsClient.find>[0],
+    'index'
+  >;
+
+  return {
+    search(searchParams: ApmAlertsClientSearchParams) {
+      return alertsClient.find({
+        index: apmAlertsIndices?.join(',') ?? '.alerts*',
```

Comment:
Do alerts end up in the .alerts* indices or .internal.alerts*?

Reply:
I'm not sure, what's the difference?

Reply:
@dgieselaar I had the same concern: 
https://github.com/elastic/kibana/pull/144484#discussion_r1018977135

@MiriamAparicio It feels problematic that we specify the default index (`.alerts*`) for another teams code here. We should use a constant like `DEFAULT_ALERTS_AS_DATA_INDEX` provided by the rac team.
This ensure that if the underlying index is changed/migrated it will still work for us.

Reply:
Should then ask rac team to add this constant?

Reply:
yes, ask the rac team what they recommend for querying alerts.